Modules/vc-module-export/VirtoCommerce.ExportModule.Web/Scripts/blades/export-settings.js

```diff
@@ -100,6 +101,10 @@ angular.module('virtoCommerce.exportModule')
 
         $scope.providerChanged = function () {
             $localStorage.defaultExportProvider = blade.selectedProvider.id;
+            resetState();
```

Comment:
Why do we need to call API again for getting the same set of types? Can't we just fill columns properly (extract method for this from getKnownTypes and call it from here)?